doc/api/process.md

```diff
@@ -1711,15 +1711,15 @@ the script name. These options are useful in order to spawn child processes with
 the same execution environment as the parent.
 
 ```bash
-node --input-type=module --eval 'console.log("foo")' script.js --version
+node --watch-path=./foo --require ./bar.js script.js --version
```

Comment:
Unfortunately watch mode affects the `execArgv`.

```console
$ echo 'console.log(process.execArgv, process.argv)' > bar.js
$ cp bar.js script.js
$ node --watch-path=./foo --require ./bar.js script.js --version
(node:33362) ExperimentalWarning: Watch mode is an experimental feature and might change at any time
(Use `node --trace-warnings ...` to show where the warning was created)
[] [
  '/nix/store/2x5s9mrmwr3myy81473r3vd1xpnd6d5m-nodejs-slim-20.10.0/bin/node',
  '/Users/duhamean/Documents/transloadit/api2/bar.js',
  'script.js',
  '--version'
]
Completed running 'script.js --version'
^C%      
```


 Let's use something more neutral:
```suggestion
node --icu-data-dir=./foo --require ./bar.js script.js --version
```